docs/documentation/helpcenter.md

```diff
@@ -163,6 +169,15 @@ Do not use `we` to refer to Zulip or its creators; for example, "Zulip also
 allows ...", rather than "we also allow ...". On the other hand, `you` is ok
 and used liberally.
 
+### Keyboard shortcuts
+
+Use backticks for each keyboard key in the shortcut (e.g. `Enter` or `R`).
+For shortcuts with more than one key, add a plus (+) surrounded by spaces
+between each key; for example `Ctrl` + `K`, instead of `Ctrl + K` or
+`Ctrl`+`K` or `Ctrl` `K`).
```

Comment:
This might be a good place to also specify that we use the labels one sees on the actual keyboard rather than the letter they produce, so "`P`" and "`Shift`+`P`" rather than "`p`" and "`P`".

(With a note that we're currently inconsistent about this, pending #21753, and that if there are already some shortcuts in the existing article then it's more important to be consistent with those than with the global style.)

Reply:
Added this as a separate paragraph that follows the one about formatting the keys with backticks.